index.html

```diff
@@ -3710,16 +3746,16 @@ <h2><span class="chunk">mDCv</span> Mastering Display Color Volume</h2>
           the Mastering Display Color Volume (mDCv) used at the point of content creation, 
           as specified in [[SMPTE-ST-2086]]. The mDCv provides informative static metadata to 
           allow a destination (consumer) display to optimize it's tone and color mappings based 
-          on its inherent capabilities. The mDCv chunk should be included for both PQ and SDR 
-          images. It is less common for HLG images. Color Primaries and White Point characteristics 
-          can be derived from cICP chunk formats. Specific examples of its most common use-cases 
-          for both HDR and SDR are available in [[ITU-T-Series-H-Supplement-19]].</p>
+          on its inherent capabilities. The mDCv chunk should be included for PQ 
+          images. It is less common for other image formats but may be included. Color Primaries and White Point characteristics 
```

Comment:
Do we have any other examples besides absolute luminance, display-referred, BT.2100 PQ?

Reply:
I'm not sure that was me ... perhaps @digitaltvguy?

Reply:
Oh, you're right. Sorry.